apps/common-app/src/examples/RuntimeTests/tests/layoutAnimations/layout/testComponents.test.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from 'react';
+import { View, StyleSheet } from 'react-native';
+import Animated from 'react-native-reanimated';
+import { useTestRef } from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+
+export const TRANSITION_REF = 'TRANSITION_REF';
```

Comment:
Technically this is not a ref but just a name of ref 😅

Reply:
You are completely right, but I was using this style a lot. I think that it is not bad to keep this misnomer, since:
* it is shorter
* it is still easy to guess that it is the ref name, not the ref, because we use UPPER_CASE syntax for hardcoded constant data.